sdk/content-components/documentation/docs/api/ContentLocationsElement.md

```diff
@@ -0,0 +1,40 @@
+---
+id: ContentLocationsElement
+title: ContentLocationsElement
+---
+
+`ContentLocationsElement` is a custom HTML element that displays a customizable SAS Viya "locations" list for navigation. Locations are typically Viya folders. This element extends <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement">`HTMLElement`</a>.
+
+## Custom Element Tag
+
+```html
+<sas-content-locations
+  url="http://my-viya-server.com"
+></sas-content-locations>
+```
+
+## Attributes
+
+### `url`
+
+See [url](LogonWrapperProps.md#url).
+
+### `authenticationType`
+
+See [authenticationType](LogonWrapperProps.md#authenticationType).
+
+## Properties
+
+### `onSelect: (item: Item) => void`
+
+A callback function that can be used to respond to the selection of a location in the ContentLocationsElement.
```

Comment:
Is "respond to" what we want to say here? Just checking whether this should actually just say "return"

Reply:
Yeah, I think so. The selection has already happened, and this just lets the consumer do something "in response" to it.